2024-09-07 16:59:58,784 - ERROR - Validation error at index 7580: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '1YMRDJNU2T', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'FTKRTQHFZE', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '7EPFWYOELT', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'DXBU6GX1VC', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `W` at 5', 'input': '3A21WYQSY7', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `W` at 5'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 16:59:58,882 - ERROR - Validation error at index 8344: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '9GN4VIO5A8', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '14NRQ80L5E', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'K4702YOYPT', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'OEOXBP8X6D', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `U` at 3', 'input': '66ULRXEWSU', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `U` at 3'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 16:59:59,319 - ERROR - Validation error at index 11873: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 16:59:59,563 - ERROR - Validation error at index 13873: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NF46JOZMTA', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '33F45BZPSP', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'ZS0PDDW72O', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '1Z18F4RM05', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `X` at 1', 'input': 'XXHOTW6WA7', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `X` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 16:59:59,618 - ERROR - Validation error at index 14280: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:00,361 - ERROR - Validation error at index 20358: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:00,857 - ERROR - Validation error at index 23525: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:00,901 - ERROR - Validation error at index 23794: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:01,485 - ERROR - Validation error at index 27347: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:01,497 - ERROR - Validation error at index 27444: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'LZLLPZ0ZUA', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'G3DEZY8UW6', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'TTH9JE93YZ', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'GT3JKF575H', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `T` at 2', 'input': '3TL8P43R9J', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `T` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:01,647 - ERROR - Validation error at index 28521: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'YULO5U0ZAM', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '0M8BGI0CI3', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'CZ35WI1011', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'CM5MTJKXMH', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 1', 'input': 'ZAJPIRYLAL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:02,105 - ERROR - Validation error at index 32353: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'SAT4V9O2DL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'O17F6WE1TD', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'PYXMXY268K', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '5OQGE7K2AV', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 1', 'input': 'RKDX2XBIJ4', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:02,524 - ERROR - Validation error at index 35767: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:03,032 - ERROR - Validation error at index 39870: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '3ZZ5UCZR5D', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '9DKC6PW41E', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'XAOROIDDK6', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '1JCRGU3GIE', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 3', 'input': '10NCUUENK0', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 3'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:03,956 - ERROR - Validation error at index 46709: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'DGQAH7M1HQ', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'I5367BRUVN', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'DEQME0YBTK', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'SQX52VSNMM', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `P` at 4', 'input': '3CEP13OI0F', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `P` at 4'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:04,188 - ERROR - Validation error at index 48592: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '4FHLELF101', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'EB2N507OZ0', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'DK93ZX02KL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'ALOGCWS9Y3', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `I` at 1', 'input': 'IW5ZTTFD6Z', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `I` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:04,480 - ERROR - Validation error at index 50988: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:04,637 - ERROR - Validation error at index 52129: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '22JSMNGJCU', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'KO7BGRPOKH', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '8XA0GSY2Z6', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '7DNU2UWFP7', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `H` at 2', 'input': 'AHXVM4Q3LH', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `H` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:04,735 - ERROR - Validation error at index 52843: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'EB8VJHYZLE', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'RA8D4CIQOV', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'Y1D3T5NOZ8', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'EOHYT5T70F', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `V` at 1', 'input': 'VKRATJXGHO', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `V` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:04,790 - ERROR - Validation error at index 53213: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '2VZEREEIKB', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'FXC3K5LZZX', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '1OFIGIX6Q9', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '5MUU1NKRED', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `I` at 1', 'input': 'I9GYGGEMVT', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `I` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:05,033 - ERROR - Validation error at index 55040: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:05,623 - ERROR - Validation error at index 59765: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'K9ZN06ZS1X', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'FIEOPTN7WZ', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '41OFB3C3PD', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '7RR8SRXQAW', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `M` at 1', 'input': 'M4G60RX0N8', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `M` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:05,777 - ERROR - Validation error at index 60950: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '9P3C0WBWTU', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'AQLUVY7DA2', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '814572Q64C', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'SSF9ANE440', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `H` at 1', 'input': 'HG2SH6D7K5', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `H` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:06,755 - ERROR - Validation error at index 67893: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:07,724 - ERROR - Validation error at index 76142: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'W6FT760O2B', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '9QDY0WMH6K', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'WRBNFJ46LJ', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': '1PZDMCME1C', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Y` at 4', 'input': 'D54Y9E2G22', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Y` at 4'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:08,516 - ERROR - Validation error at index 82628: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:08,738 - ERROR - Validation error at index 84279: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'DOIR43VTCM', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '5RZL03AWR6', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'PKTU8OYDWF', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'KQVJ34AINL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 1', 'input': 'OQ0D67INSG', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:09,487 - ERROR - Validation error at index 90453: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:09,663 - ERROR - Validation error at index 91807: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'FA8KD82QH3', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'QF6S8TDTEA', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'G3Z42NGBRE', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'QA65EOIBX4', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 2', 'input': 'CZ4ZQDXNZC', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:09,687 - ERROR - Validation error at index 91998: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '03T414PVFI', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'L1N4X0SVZA', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'XZJTZGW546', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'YRYN6Y8SPJ', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 3', 'input': 'FFRVR03VV3', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 3'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:09,751 - ERROR - Validation error at index 92418: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'FNPZFYI489', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'OVDJZCARJA', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '83QG97URRC', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'JMW951JPZC', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 2', 'input': '7OYKS8WHL6', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:09,937 - ERROR - Validation error at index 93810: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '67RMH5U2R6', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'UDHIYJS2GP', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '8GZBM34AGM', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'DZC37NLW4F', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 1', 'input': 'OEIZJ5ZFEX', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `O` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:10,210 - ERROR - Validation error at index 95912: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:10,547 - ERROR - Validation error at index 98710: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'J9VQLERJQO', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'LND1WX0Y6Z', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'W5A22P7J9N', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'SYID3PBQLP', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 2', 'input': '0ZP1AQ1828', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `Z` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:11,066 - ERROR - Validation error at index 102999: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:11,163 - ERROR - Validation error at index 103671: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'ZRH2YT3FR8', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': '3GJWN253MM', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'K179M0CI4M', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'IXNB2XXEKB', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 3', 'input': 'BCRRBKY2PD', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 3'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:11,627 - ERROR - Validation error at index 107189: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'GYSATSCN88', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'M9ZV3N8G95', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'WRTW643A1S', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'MZIS9E7IXD', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 2', 'input': '7R2EEPHB6J', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `R` at 2'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
2024-09-07 17:00:11,645 - ERROR - Validation error at index 107326: [{'type': 'int_parsing', 'loc': ('month',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('year',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'int_parsing', 'loc': ('day',), 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': 'NULL', 'url': 'https://errors.pydantic.dev/2.8/v/int_parsing'}, {'type': 'literal_error', 'loc': ('time_period',), 'msg': "Input should be 'Morning', 'Midday', 'Evening' or 'Late_Hours'", 'input': 'NULL', 'ctx': {'expected': "'Morning', 'Midday', 'Evening' or 'Late_Hours'"}, 'url': 'https://errors.pydantic.dev/2.8/v/literal_error'}, {'type': 'uuid_parsing', 'loc': ('date_uuid',), 'msg': 'Input should be a valid UUID, invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1', 'input': 'NULL', 'ctx': {'error': 'invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `N` at 1'}, 'url': 'https://errors.pydantic.dev/2.8/v/uuid_parsing'}]
